sensibility/miner/downloader.py

```diff
@@ -200,12 +202,15 @@ def fetch_repository(self, repo: RepositoryID) -> RepositoryMetadata:
             raise ValueError(f'Could not fetch info for {repo}')
         owner, name = RepositoryID.parse(info['nameWithOwner'])
         latest_commit = info['defaultBranchRef']['target']
+        licenseName = None
```

Comment:
Two issues here:

 1. please use `snake_case` to name variables, as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names): that is,  `license_name`
2. This variable should be declared as `str` or `Optional[str]`

Reply:
sounds good! will do.